src/jdk.jlink/share/classes/jdk/tools/jlink/resources/plugins.properties

```diff
@@ -70,14 +70,20 @@ An optional <pattern-list> filter can be specified to list the pattern of\n\
 files to be included.
 
 compress.usage=\
-\  --compress <0|1|2>[:filter=<pattern-list>]\n\
+\  --compress <0|1|2|zip-[0-9]>[:filter=<pattern-list>]\n\
```

Comment:
This is a bit odd - when I run the `jlink`  command against latest mainline, I don't see the `[:filter=<pattern-list>]` being displayed in the help output nor do I see the message which states `An optional <pattern-list> ....`. This is what the `jlink --help` looks like against current mainline:
(first check the version)
```
jlink --version
21-internal
```
(check help output)
```
jlink --help
Usage: jlink <options> --module-path <modulepath> --add-modules <module>[,<module>...]
Possible options include:
      --add-modules <mod>[,<mod>...]    Root modules to resolve in addition to the
                                        initial modules. <mod> can also be ALL-MODULE-PATH.
      --bind-services                   Link in service provider modules and
                                        their dependences
  -c, --compress=<0|1|2>                Enable compression of resources:
                                          Level 0: No compression
                                          Level 1: Constant string sharing
                                          Level 2: ZIP
```

So it appears that this is not where the `-c --compress` option's help text is being picked up from?

Reply:
Irrespective of this issue, I believe the `compress.argument` and `compress.description` property values in this file would need an update too.